tests/console/sysstat.pm

```diff
@@ -35,7 +35,7 @@ sub run {
     assert_script_run '/usr/lib64/sa/sa1 5 5';
 
     #Set 24h clock(removes AM/PM extra column), extract a pid number, confirm its an integer
-    validate_script_output "LC_TIME='C' pidstat  |awk '{print \$3}' |head |tail -n 1", sub { m/[1-999]/ };
+    validate_script_output "LC_TIME='C' pidstat  |awk '{print \$3}' |head |tail -n 1", sub { /\d+/ };
```

Comment:
It fixes regexp, but does it verify value `is` integer or `contains` integer? Cause second case returns true for any column returned by pidstat:
```yaml
22:47:26        0        10    0,00    0,00    0,00    0,00    0,00     0  migration/0
```

I am just guessing, but I think it should be `/^\d+$/`

Reply:
@kravciak  initial regexp also not have this extra conditions , and I think that they not needed because of `awk` we already filter whole string except 3d column so all should be fine from this perspective. 
But another thing bothers me - initial regexp was more strict , it was saying that pid should be below 1000 and now it could be any value even above 65536 so we lower testing coverage here . if it is about only zero why not simply do : 
```suggestion
    validate_script_output "LC_TIME='C' pidstat  |awk '{print \$3}' |head |tail -n 1", sub { /[0-999]/ };
```

Reply:
@kravciak I had the same thought when I looked at it initially, and thought it was a false negative because of that, but on s390 for example, the 3rd column (pid's) gives 0's and as cherry on top: `\n` :D `\d/+` will match digits.

We could go even further and try to verify if the returned value is actually a valid pid... but i'd be beyond what this pr is addressing...

Reply:
@asmorodskyi https://regex101.com/ ;) - I use it often, see what [1-999] means

@foursixnine I agree it's improvement, feel free to merge.
So if I get it right: every column in output contains some digit, this verifies that pidstat produces any output, right?

Reply:
@kravciak  ah so regex was really misleading , I thought it covers interval, so I think I will just merge this one it would be better in any case